deployment/refinery-modules/refinery/manifests/postgresql.pp

```diff
@@ -0,0 +1,32 @@
+class refinery::postgresql {
+
+  if $::deployment_platform == 'aws' {
```

Comment:
This is just a global variable that we set, like `$::app_user` for example. The plan is to replace all the globals with class parameters to make classes self-contained.